ansys/dpf/core/field_definition.py

```diff
@@ -105,6 +105,21 @@ def dimensionality(self, value):
     def _modify_field_def(
         self, unit=None, location=None, dimensionality=None, shell_layer=None
     ):
+    def deep_copy(self, server=None):
```

Comment:
@cbellot000 is the `deep_copy` function supposed to be inside the `_modify_field_def` function?